packages/components/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 <!-- Learn how to maintain this file at https://github.com/WordPress/gutenberg/tree/HEAD/packages#maintaining-changelogs. -->
 
 ## Unreleased
--	`Dropdown` : Add styling support for `MenuGroup` ([#59723](https://github.com/WordPress/gutenberg/pull/59723)).
--	`Placeholder` : Allow overflow but only when placeholder is selected, to fix a layout shift. `MenuGroup` ([#59857](https://github.com/WordPress/gutenberg/pull/59857)).
+
+-   `Dropdown` : Add styling support for `MenuGroup` ([#59723](https://github.com/WordPress/gutenberg/pull/59723)).
+-   `Placeholder` : Allow overflow but only when placeholder is selected, to fix a layout shift. `MenuGroup` ([#59857](https://github.com/WordPress/gutenberg/pull/59857)).
+-   `Utils` : Update the line height in the Javascript to match to CSS.
```

Comment:
The utils file is not even publicly exported, so it's more like we want to alert the consumers of the affected components for possible style breakage. So something like:

```suggestion
-   `Text`, `Heading`, `ItemGroup` : Update the line height from 1.2 to 1.4.
```

And I think this would be categorized under Enhancements. (Not sure why there are uncategorized items listed 🤷)